homeassistant/components/zwave_js/device_action.py

```diff
@@ -300,6 +325,15 @@ async def async_get_action_capabilities(
             )
         }
 
+    if action_type == SERVICE_GET_LOCK_USERCODE:
+        return {
+            "extra_fields": vol.Schema(
+                {
+                    vol.Required(ATTR_CODE_SLOT): cv.string,
+                }
+            )
+        }
```

Comment:
you'd have to update the device action tests to cover this. There are multiple tests that likely need to be updated in the `test_device_action` model to cover this additional service

Reply:
you can also pull out the device action changes instead and submit separately, I don't think we are actually at parity with the new services for device actions so you could include those other ones. Just a thought, not a requirement